src/Raven.Server/ServerWide/Commands/ModifyDatabaseWatchersCommand.cs

```diff
@@ -64,14 +65,22 @@ public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
             if (Watcher == null)
                 return null;
 
-            var newTaskKey = Watcher.GetTaskKey().ToString();
-            if (Watcher.CurrentTaskId != null && Watcher.CurrentTaskId != newTaskKey)
+            var newTaskId = (Watcher.Database + Watcher.Url).ToLower();
```

Comment:
ok, but ther'es another problem about removing the task id. after discussing about it with @Danielle9897 and @grisha-kotler (see comment here: http://issues.hibernatingrhinos.com/issue/RavenDB-7196 ), we decided that we should create GetOngoingTaskInfo and DeleteOngoingTask endpoints that receives taskId and TaskType. Beacuse of that we think that all ongoing-tasks should have a taskId